about.html

```diff
@@ -80,11 +151,12 @@ <h2>Partners</h2>
                 <li><img src="./img/daum.png" alt="partners copany"></li>
                 <li><img src="./img/airbnb.png" alt="partners copany"></li>
             </ul>
-            </div>
-    </section> 
+        </div>
+    </section>
     <footer>
```

Comment:
- [ ]  Just as the previous reviewer mentioned, kindly make your footer have a dark background, also the partner section is not suppose to be seen in your about.html desktop view
your design :point_down: 
      
![image](https://user-images.githubusercontent.com/66097371/164648721-fc747b76-a7c6-4314-bec5-85e2043a14ef.png)
project design :point_down: 
![image](https://user-images.githubusercontent.com/66097371/164648899-64f01877-5742-4460-a21a-9bfd6b5e26d8.png)